src/tests/components/ErrorBoundary.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import { ErrorBoundary } from '../../components/UI'
+
+const ErrorText = 'Error message'
+const ErrorChild = <div className='custom-class'>{ErrorText}</div>
+
+describe(`ErrorBoundary component test`, () => {
+  it('should component rendered properly with status [false]', () => {
+    const rendered = render(
+      <ErrorBoundary showError={false}>{ErrorText}</ErrorBoundary>,
+    )
+    expect(rendered).toBeDefined
+  })
+  it('should component rendered properly with status [true]', () => {
+    const rendered = render(
+      <ErrorBoundary showError={true}>{ErrorText}</ErrorBoundary>,
+    )
+    expect(rendered).toBeDefined
+  })
+  it('should component rendered properly with status [true] and with H1 tag', () => {
+    const rendered = render(
+      <ErrorBoundary showError={true}>
+        <h1>{ErrorText}</h1>
+      </ErrorBoundary>,
+    )
+    expect(rendered).toBeDefined
```

Comment:
You can try to find a `h1` element to make the test a bit more accurate. 

For example:

```
const { container } = render(
      <ErrorBoundary showError={true}>
        <h1>{ErrorText}</h1>
      </ErrorBoundary>,
    )
    
    // Find by selector:
    const h1Element = container.querySelector('h1')
    expect(h1Element).toBeTruthy()
    
    // ...or find by text:
    const h1Element = container.getByText(ErrorText)
    expect(h1Element).toBeTruthy()
```

Reply:
I will change it after merge because there is more file which needs to be changed